MobileWallet/Common/NotificationManager.swift

```diff
@@ -54,22 +85,51 @@ class NotificationManager {
         }
 
         notificationCenter.requestAuthorization(options: options) {
-            (didAllow, error) in
+            (_, error) in
             guard error == nil else {
                 TariLogger.error("NotificationManager request authorization", error: error)
+
                 completionHandler(false)
                 return
             }
 
-            completionHandler(didAllow)
+            self.registerWithAPNS(completionHandler)
         }
     }
 
-    func checkAuthorization(_ completionHandler: @escaping (Bool) -> Void) {
+    //If a previous attempt to register the device's token failed, this will retry
+    func retryIfTokenNotRegistered() {
+        //If they don't have a wallet yet, don't even try as there is no pub key to register
+        guard TariLib.shared.tariWallet != nil else { return }
+
+        if !hasRegisteredToken {
+            requestAuthorization { (_) in }
+        } else {
+            TariLogger.verbose("Already registered for push notifications")
+        }
+     }
+
+    private func registerWithAPNS(_ completionHandler: @escaping (Bool) -> Void) {
+        //If we don't have the server deployed yet
+        guard !TariSettings.shared.pushNotificationServer.isEmpty else {
+            TariLogger.warn("No push notification server setup yet")
+            completionHandler(true)
```

Comment:
Shouldn't it be `false`?

Reply:
Reason I made it true was because I didn't want it reporting errors as I thought I needed to deploy a testflight before we had the push server deployed. We know have the push server so I'm actually going to remove this check all together, no longer needed